WalletWasabi.Fluent/Views/AddWallet/WelcomePageView.axaml

```diff
@@ -243,7 +243,7 @@
           </Viewbox>
           <DockPanel Grid.Column="0" Grid.Row="0" VerticalAlignment="Center">
             <TextBlock Classes="text"
-                       Text="Wasabi provides privacy by default, it protects your financial information through coinjoins, client-side block filtering and communication over the Tor anonymity network.&#10;&#10;Neither the public nor the developers can breach your privacy." />
+                       Text="Wasabi provides privacy by default.&#10;&#10;Nobody can breach your privacy.&#10;&#10;And for that you pay a fee to the Wasabi backend." />
```

Comment:
They have zero ideas what is a backend.

Reply:
The important thing is they pay a fee, the other less important aspect is whom the fee goes to.

I guess the latter can be removed?

Reply:
I agree